lib/auth.coffee

```diff
@@ -62,7 +80,12 @@ getAuth = (deps, opts) ->
 	# });
 	###
 	exports.whoami = (callback) ->
-		token.getUsername().asCallback(callback)
+		exports.getUserDetails()
+			.then (body) ->
```

Comment:
NIT: Most chained method calls in the SDK have the same indentation as their previous statement. Please reduce the indentation by 1 level.
So ideally this should be
```
exports.getUserDetails()
.then (body) ->
```
In other words, regardless of whether the `.then` is on top or in a new line, the indentation of the code of the inner function ends up to be the same.